packages/pigeon/lib/swift_generator.dart

```diff
@@ -125,7 +125,7 @@ class SwiftGenerator extends StructuredGenerator<SwiftOptions> {
         indent, classDefinition.documentationComments, _docCommentSpec,
         generatorComments: generatedComments);
 
-    indent.write('struct ${classDefinition.name} ');
+    indent.write('struct ${classDefinition.name}: Sendable ');
```

Comment:
Can you create a dart annotation to gate this? this will likely crash older swift versions